apps/pearl-api/pages/web3auth/login.tsx

```diff
@@ -25,11 +26,10 @@ const Web3AuthModal = () => {
 
   useEffect(() => {
     // Notify once initialized
-    if (isInitialized)
-    window.parent.postMessage({
-      event_id: Events.WEB3AUTH_MODAL_INITIALIZED
-    }, '*');
-  }, [isInitialized])
+    if (isInitialized && typeof window !== 'undefined') {
```

Comment:
are these window checks required for types? because the hooks would only run when window is present 🤔